build.sbt

```diff
@@ -9,18 +9,29 @@ lazy val commonSettings = Seq(
 
   scalaVersion := "2.13.14",
   Compile / doc / scalacOptions ++= Seq("-groups", "-implicits", "-no-link-warnings"),
-  scalacOptions := Seq("-release:17", "-Xsource:3-cross", "-deprecation"),
+  scalacOptions := Seq("-release:17", "-Xsource:3", "-deprecation"),
   releaseCrossBuild := true,
   publishTo := Some(Resolver.evolutionReleases),
+  versionPolicyIntention := Compatibility.BinaryCompatible, // sbt-version-policy
   versionScheme := Some("semver-spec"),
 
   /*testOptions in Test ++= Seq(Tests.Argument(TestFrameworks.ScalaTest, "-oUDNCXEHLOPQRM"))*/
   libraryDependencies += compilerPlugin(`kind-projector` cross CrossVersion.full),
 
+  // TODO remove after 4.0.7 is released
+  versionPolicyIgnored := Seq(
+    "com.evolutiongaming" %% "random", // removed as was not used
```

Comment:
I'm bit confused but when/how 
```
"com.evolutiongaming" %% "random"
```
was removed? I do not see it in the MR (might be im just blind/still on holiday)?

Reply:
mima plugin claims, that 4.0.6 depends on `random` 🤷

Reply:
Yes, `retry` depends on `random` so its expected. My point is that `random` _was not removed_ from dependencies: its still there as transitive one. Now i see in master 
```
com.evolutiongaming:random_2.13:1.0.0
```
What is `random` version in the branch? I would like to see why its not binary compatible

Reply:
it is `1.0.4` on my branch

Reply:
`retry 3.1.0` depends on `random 1.0.4`

Reply:
I've checked `random` changes and looks like its safe to ignore MiMa warnings about it, here's the diff: https://github.com/evolution-gaming/random/compare/v1.0.0...v1.0.4